### PR TITLE
Nc games 8

### PR DESCRIPTION
**Pull request by SZahl**

### CODE REVIEW

File: app.models.js [hunk 1]
```diff
@@ -63,4 +63,17 @@ exports.checkUserExists = (id) => {
             return Promise.reject({ message: 'Username not found', status: 404 })
         }
     })
+}
+
+exports.amendVoteCount = (requestBody, id) => {
+    const { inc_votes } = requestBody;
+    if (typeof inc_votes !== 'number' || !inc_votes) {
```
Comment (NRMSMITH) [inline app.models.js]: psql will throw an error if it is not a number :)

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
PSQL should throw an error if the article_id is not a number, so a promise.reject isn't relevant for it!

File: app.models.js (hunk 1 above)
Comment (SZahl) [reply app.models.js]: Updated. Am I good to merge now?

**Review by NRMSMITH — APPROVED** [review]
🚀

**Pull request merged**